.github/workflows/build.yml

```
@@ -95,7 +95,7 @@ jobs:
       matrix:
         # x64=x86_64, evansport=i686, aarch64=armv8, armv7, hi3535=armv7l, 88f6281=armv5, qoriq=ppc
         # https://github.com/SynoCommunity/spksrc/wiki/Synology-and-SynoCommunity-Package-Architectures
-        arch: [noarch, noarch-6.0, noarch-7.0, x64-6.1, x64-7.0, evansport-6.1, evansport-7.0, aarch64-6.1, aarch64-7.0, armv7-6.1, armv7-7.0, hi3535-6.1, 88f6281-6.1, qoriq-6.1]
+        arch: [noarch, noarch-6.0, noarch-7.0, x64-6.1, x64-7.0, evansport-6.1, evansport-7.0, aarch64-6.1, aarch64-7.0, armv7-6.1, armv7-7.0, hi3535-6.1, 88f6281-6.1, qoriq-6.1, comcerto2k-6.1, comcerto2k-7.0]
```

comcerto2k-6.1 is still contained in `arch-armv7-6.1`, so we only need this for DSM 7.
```suggestion
        arch: [noarch, noarch-6.0, noarch-7.0, x64-6.1, x64-7.0, evansport-6.1, evansport-7.0, aarch64-6.1, aarch64-7.0, armv7-6.1, armv7-7.0, hi3535-6.1, 88f6281-6.1, qoriq-6.1, comcerto2k-7.0]
```

---

Yes, but I'm not sure if it works, I don't understand fully how the framework is filtering it out:

---

supported archs are put in the INFO file in the packages and are taken from the TC-Makefile (`toolchain/syno-armv7-6.1/Makefile` and `toolchain/syno-armv7-7.0/Makefile` for generic armv7) and the spksrc repo takes the info from that file on publish too.

---

So as we keep comcerto2k in the armv7-6.1 you cannot publish a package of the same SPK-Version for comcerto2k-6.1 and armv7-6.1.

---

Thanks

> you cannot publish a package of the same SPK-Version for comcerto2k-6.1 and armv7-6.1.

:thinking:  That could be a problem.